python_dwd/parsing_data/parse_data_from_files.py

```diff
@@ -4,14 +4,16 @@
 from io import BytesIO
 import pandas as pd
 
-from python_dwd.additionals.helpers import create_stationdata_dtype_mapping
+from python_dwd.additionals.helpers import create_stationdata_dtype_mapping, convert_datetime_hourly
 from python_dwd.constants.column_name_mapping import GERMAN_TO_ENGLISH_COLUMNS_MAPPING
 from python_dwd.constants.metadata import NA_STRING, STATIONDATA_SEP
+from python_dwd.enumerations.column_names_enumeration import DWDMetaColumns
+from python_dwd.enumerations.time_resolution_enumeration import TimeResolution
 
 log = logging.getLogger(__name__)
 
 
-def parse_dwd_data(filenames_and_files: List[Tuple[str, BytesIO]]) -> pd.DataFrame:
+def parse_dwd_data(filenames_and_files: List[Tuple[str, BytesIO]], time_resolution: TimeResolution) -> pd.DataFrame:
```

Comment:
Change to : `time_resolution: Optional[str, TimeResolution]` 
And add: 
`time_resolution = TimeResolution(time_resolution)`
Because this function is also used as a public service

Reply:
I've modified the parameter within the function signature to `time_resolution: Union[TimeResolution, str]`. I believe this is what you suggested?